src/components/pages/home/homepage-data.ts

```diff
@@ -214,12 +214,11 @@ There are a number of tried and true methods that will allow you to learn more e
         path: '/courses/practical-git-for-everyday-professional-use',
       },
       {
-        title: 'Develop Accessible Web Apps with React',
-        byline: 'Erin Doyle・88m・Course',
+        title: 'Build an App with React Suspense',
+        byline: 'Michael Chan・1h 23m・Course',
```

Comment:
we already have this course on the page so I switched it up for a different one

Reply:
This makes sense!